trpc/queries/goalV2.ts

```diff
@@ -59,6 +57,10 @@ const mapSortParamsToTableColumns = (sort: QueryWithFilters['sort']): Array<Orde
     });
 };
 
+/**
+ * @deprecated
+ * DONT USE THIS QUERY
+ */
 export const getGoalList = (params: GoalQueryParams) => {
```

Comment:
Can we remove it? I can't find usage.

Reply:
Maybe not in this pr?)

Its will use in future